src/dropdown/dropdown.component.ts

```diff
@@ -387,36 +387,36 @@ export class Dropdown implements OnInit, AfterContentInit, AfterViewInit, OnDest
 		const isUpdate = event => event && event.isUpdate;
 
 		this.view.select.subscribe(event => {
-			if (this.type === "single" && !isUpdate(event)) {
-				this.closeMenu();
-				if (event.item && event.item.selected) {
-					if (this.itemValueKey) {
-						this.propagateChange(event.item[this.itemValueKey]);
+			this.ngZone.run(() => {
```

Comment:
```suggestion
			setTimeout(() => {
```

Reply:
Unfortunately, using `setTimeout` does resolve the issue highlighted in https://github.com/carbon-design-system/carbon-components-angular/issues/802.